sa/saro.go

```diff
@@ -1266,3 +1266,122 @@ func (ssa *SQLStorageAuthorityRO) GetPausedIdentifiers(ctx context.Context, req
 
 	return newPBFromIdentifierModels(matches)
 }
+
+// GetRateLimitOverride retrieves a rate limit override for the given bucket key
+// and limit. If no override is found, a NotFound error is returned.
+func (ssa *SQLStorageAuthorityRO) GetRateLimitOverride(ctx context.Context, req *sapb.GetRateLimitOverrideRequest) (*sapb.RateLimitOverrideResponse, error) {
+	if core.IsAnyNilOrZero(req, req.LimitEnum, req.BucketKey) {
+		return nil, errIncompleteRequest
+	}
+
+	var row overrideModel
+	err := ssa.dbReadOnlyMap.SelectOne(ctx, &row,
+		`SELECT * FROM overrides
```

Comment:
We try to avoid "SELECT *"; see https://github.com/letsencrypt/boulder/issues/4909. I think it would be better to a) list the column names explicitly, or b) rely on the fact that you told the database which columns form the primary key in database.go and use borp's built-in dbMap.Get() method to do its magic.